webapp/channels/src/components/cloud_preview_modal/cloud_preview_modal.tsx

```diff
@@ -19,26 +24,33 @@ const CloudPreviewModal: React.FC = () => {
     const subscription = useSelector(getCloudSubscription);
     const license = useSelector(getLicense);
     const currentUserId = useSelector(getCurrentUserId);
+    const team = useSelector(getCurrentTeam);
 
     const isCloud = license?.Cloud === 'true';
     const isCloudPreview = subscription?.is_cloud_preview === true;
 
     // Check if modal has been shown before
-    const hasModalBeenShown = false;
+    const hasModalBeenShown = useSelector((state: GlobalState) => getPreference(state, CLOUD_PREVIEW_MODAL_SHOWN_PREF, CLOUD_PREVIEW_MODAL_SHOWN_PREF)) === 'true';
```

Comment:
Nit: This could use the `getBool` selector instead of the `get` so that you don't need the explicit equality check